server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -6751,8 +6737,23 @@ private boolean isImplicitPlannerUsedByOffering(long offeringId) {
         return implicitPlannerUsed;
     }
 
-    private boolean isVmVolumesOnZoneWideStore(VMInstanceVO vm) {
-        final List<VolumeVO> volumes = _volsDao.findCreatedByInstance(vm.getId());
+    private boolean isVmVolumesUsingLocalStorage(final List<VolumeVO> volumes) {
```

Comment:
The code in this method looks exactly like the bit in `isVmVolumesOnZoneWideStore`